src/Shared/Recruit.Shared.Web/Models/VacancyAction.cs

```diff
@@ -0,0 +1,16 @@
+using System;
```

Comment:
I don't think this class is used.

Reply:
have removed and addressed the file naming mismatch. Will address the other issue you raised regarding the error on editing a draft or referred vacancy with a revoked permission when a decision is made by management

Reply:
with the decisions that have been made for:

- https://skillsfundingagency.atlassian.net/browse/ER-1081
- https://skillsfundingagency.atlassian.net/browse/ER-1148

We are no longer allowing the permission to be revoked for private beta. I have removed the check around permissions check for cloning vacancy as it is redundant. We may have to bring it back in or improve on the general set of checks that will be required when the revoke permission functionality for Recruitment is enabled.